gov.pnnl.goss.gridappsd/applications/fncsgossbridge/apps/fncs_goss_bridge.py

```diff
@@ -60,63 +31,85 @@
 isInitialized = False 
 simulationId = None
 
```

Comment:
PEP-8 states there should be 2 empty lines between the start of a class and the previous variable declaration.